src/styles/_catalog.scss

```diff
@@ -1,22 +1,32 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: 48px;
+  justify-content: center;
+  align-items: center;
+
+  grid-template-columns: repeat(4, 200px);
+  grid-auto-flow: row;
+
+
+  @media (max-width: 1024px) {
+    grid-template-columns: repeat(3, 200px);
```

Comment:
in the task ' - Must have four columns starting with `1024px` .  You made it will be three column starting with 1024 and below

Reply:
Thank you, I totally forgot to change breakpoints according to limits